sdk/servicebus/azure-messaging-servicebus/README.md

```diff
@@ -151,9 +197,18 @@ Example of receiving a message asynchronously is documented [here][sample-receiv
 ### Send message with Azure Active Directory credentials
 Example of sending a message asynchronously using active directory credential is documented [here][sample-send-async-aad-message].
 
+### Send a batch of messages 
```

Comment:
This is a lot of headers. It could be condensed into a list of links.
```
* [Sending a batch of messages][sample-send-batch-messages]
* [Receive messages and settle them][ ... ]
...
```